pdns/ws-recursor.cc

```diff
@@ -382,10 +382,10 @@ static void apiServerCacheFlush(HttpRequest* req, HttpResponse* resp) {
   DNSName canon = apiNameToDNSName(req->getvars["domain"]);
   bool subtree = (req->getvars.count("subtree") > 0 && req->getvars["subtree"].compare("true") == 0);
 
-  int count = broadcastAccFunction<uint64_t>(std::bind(pleaseWipeCache, canon, subtree, 0xffff));
-  count += broadcastAccFunction<uint64_t>(std::bind(pleaseWipePacketCache, canon, subtree, 0xffff));
-  count += broadcastAccFunction<uint64_t>(std::bind(pleaseWipeAndCountNegCache, canon, subtree));
-  resp->setBody(Json::object {
+  int count = broadcastAccFunction<uint64_t>(boost::bind(pleaseWipeCache, canon, subtree, 0xffff));
```

Comment:
why the change from `std` to `boost`?

Reply:
I honestly have no idea where this change came from! I don't remember changing it nor can I see any reason why it would have been changed as it's not related to the topic of this PR at all.

Reply:
Probably happened during a rebase then. Can you take it out? :)

Reply:
Done.